build.sbt

```diff
@@ -37,13 +37,14 @@ addCommandAlias(
   ";zioCacheNative/test:compile"
 )
 
-val zioVersion = "1.0.12"
+val zioVersion = "1.0.16"
 
 lazy val root = project
   .in(file("."))
   .settings(
     publish / skip := true,
-    unusedCompileDependenciesFilter -= moduleFilter("org.scala-js", "scalajs-library")
+    unusedCompileDependenciesFilter :=
+      unusedCompileDependenciesFilter.value - moduleFilter("org.scala-js", "scalajs-library")
```

Comment:
I had to change this. I'm not sure why. Probably because I update sbt.
Without this change, we were having this issue:
```scala
/home/runner/work/zio-cache/zio-cache/build.sbt:46: error: No implicit for Remove.Value[explicitdeps.ModuleFilter, sbt.librarymanagement.ModuleFilter] found,
  so sbt.librarymanagement.ModuleFilter cannot be removed from explicitdeps.ModuleFilter
    unusedCompileDependenciesFilter -= moduleFilter("org.scala-js", "scalajs-library")
                                    ^
sbt.compiler.EvalException: Type error in expression
[error] sbt.compiler.EvalException: Type error in expression
[error] Use 'last' for the full log.
```